projects/blink/src/lib/containers/blink-container/blink-container.component.ts

```diff
@@ -1,52 +1,41 @@
 import {ChangeDetectionStrategy, Component, OnInit} from '@angular/core';
-import {LightComponent, PageLayoutComponent} from '../../../../../ui/src/public-api';
-import {BlinkNavPanelContainerComponent} from '../blink-nav-panel-container/blink-nav-panel-container.component';
-import {select, Store} from '@ngrx/store';
+import {PageLayoutComponent} from '../../../../../ui/src/public-api';
+import {Store} from '@ngrx/store';
 import {ActivatedRoute} from '@angular/router';
-import {blinkSendMessage, setDeviceName} from '../../+state/blink-config/blink-config.actions';
+import {setDeviceName} from '../../+state/blink-config/blink-config.actions';
 import {RouterOutlet} from '@angular/router';
-import { BlinkAboutContainerComponent } from '../blink-about-container/blink-about-container.component';
+import { getMode } from '../../+state/blink-mode/blink-mode.selectors';
+import { BlinkModeState } from '../../+state/blink-mode/blink-mode.reducer';
+import { AsyncPipe } from '@angular/common';
+import { BlinkTwoContainerComponent } from '../blink-two-container/blink-two-container.component';
+import { BlinkThreeContainerComponent } from '../blink-three-container/blink-three-container.component';
 
 @Component({
   selector: 'lib-blink-container',
   standalone: true,
   imports: [
     PageLayoutComponent,
-    BlinkNavPanelContainerComponent,
-    LightComponent,
     RouterOutlet,
-    BlinkAboutContainerComponent
+    AsyncPipe,
+    BlinkTwoContainerComponent,
+    BlinkThreeContainerComponent,
   ],
   templateUrl: './blink-container.component.html',
   styles: ``,
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class BlinkContainerComponent implements OnInit {
+
+  mode$ = this.store$.select(getMode);
+
   constructor(
     private readonly store: Store,
-    private route: ActivatedRoute
-  ) {
+    private route: ActivatedRoute,
+    private store$: Store<BlinkModeState>){
```

Comment:
private store$: Store<BlinkModeState> to     private store$: Store

Reply:
add coma

Reply:
private store$: Store<BlinkModeState> is better